docs/tutorials/workshops/watcher/README.md

```diff
@@ -0,0 +1,154 @@
+# Watcher
+
+## Overview
+
+In this tutorial we will explore using the watcher sdk to create a watcher to
+listen for events from the EPR server.
+
+This tutorial depends on the [Hello World](../hello_world/README.md) being
+completed.
+
+## Create a new watcher
+
+Make a new directory for your watcher and create a `main.go` in that directory.
+
+```bash
+mkdir foo
+cd foo
+touch main.go
+```
+
+Now open the `main.go` in your favorite editor (Vim).
+
+Add the following code:
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/sassoftware/event-provenance-registry/pkg/message"
+	"github.com/sassoftware/event-provenance-registry/pkg/watcher"
+)
+
+func main() {
+	seeds := []string{"localhost:19092"}
+	topics := []string{"epr.dev.events"}
+	consumerGroup := "watcher-workshop"
+
+	watcher, err := watcher.New(seeds, topics, consumerGroup)
+	if err != nil {
+		panic(err)
+	}
+	defer watcher.Client.Close()
+
+	go watcher.StartTaskHandler(customTaskHandler)
+
+	watcher.ConsumeRecords(customMatcher)
+}
+
+func customMatcher(record *watcher.Record) bool {
+	var msg message.Message
+	err := json.Unmarshal(record.Value, &msg)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return msg.Type == "foo.bar"
+}
+
+func customTaskHandler(record *watcher.Record) error {
+	log.Default().Printf("I received a task with value '%s'", record.Value)
+	return nil
+}
+
+```
+
+Save the file and run `go mod init` then
+
+Add the following line to the `go.mod` file under the module line.
+
+```bash
+replace github.com/sassoftware/event-provenance-registry => ../../../../../
+
+```
+
+File should look like this:
+
+```bash
+module github.sas.com/sassoftware/event-provenance-registry/docs/tutorials/workshops/watcher/foo
+
+replace github.com/sassoftware/event-provenance-registry => ../../../../../
+
+go 1.21.4
+```
+
+Now we can run `go mode tidy` to fill in our dependencies.
+
```

Comment:
Once the repo is marked public the go imports will work as is, so we don't need to do any of the `replace` magic

```suggestion
Now we can run `go mod tidy` to fill in our dependencies.
```